### DISCUSSION TITLE
Open Source

### DISCUSSION THREAD

**Discussion by thebetioplane** [thebetioplane/osuReplayEditorV3#42] [Announcements]
I have released the source code of this application under GPLv3. It's in a different repository found here:

https://github.com/thebetioplane/osuReplayEditorV3_SRC

**Comment by Hazuki-san** [comment]
This will be only made for major releases, yes?

**Comment by thebetioplane** [comment]
The master branch of the _SRC repo is currently up-to-date with the binaries distributed here. They will be kept in sync so that way the corresponding source code for the version you can download is always available.
